github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1881,6 +1881,42 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    optional: true
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
```

Comment:
Do we need this?

Reply:
Probably not. The idea is to provide early feedback to contributors and to sort of foramalize
clean code standards in kubevirt. We don't need to report to the dashboard. [Removed](https://github.com/kubevirt/project-infra/compare/50af0d89e409b40261dddee10c90afe9299efe13..498d2712f9cba2c5ea9647c5cd468cd14f3b0ded) the annotation.